admin/test/test_installer.py

```diff
@@ -502,13 +458,23 @@ def upload_docker_compose_files():
             steps=repeat(1, 60)
         )
 
-        # This isn't in the tutorial, but docker-compose doesn't retry failed
-        # pulls and pulls fail all the time.
+        def cleanup_container(ignored):
+            self.addCleanup(
+                remote_docker_compose,
+                self.client_node_ip,
+                self.docker_host,
+                self.compose_node1.path,
+                'down'
```

Comment:
This is a new command which both stops and removes the containers....which simplifies things here and in the docs (later).